README.md

```diff
@@ -1,239 +0,0 @@
-<!-- markdownlint-disable-file MD033 - inline HTML is allowed here -->
```

Comment:
Without a `README.md` file at the root, Github will show `docs/README.md` on the front page by default. But that file has YAML front matter (which Github doesn't support and displays as a table) and may include other [ReadMe-Flavored Markdown](https://rdmd.readme.io/docs) features in the future.

I think it's beneficial to keep a top-level README as a polished front page of the Github repository even if it's just a pretty logo and a link to our docs on `readme.com`

Reply:
Added polished README to top-level and renamed the version in /docs to `quickstart.md` to match the user docs.  The document is identical except the front matter and we will just need to keep README updated as needed.